packages/components/psammead-section-divider/README.md

```diff
@@ -0,0 +1,60 @@
+# psammead-section-divider - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-section-divider%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-section-divider%2Fpackage.json) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/section-divider--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-section-divider.svg)](https://www.npmjs.com/package/@bbc/psammead-section-divider) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `@bbc/psammead-section-divider` package exports a single Section Divider component.
+
+It uses `@bbc/psammead-styles` for colours and font family and `@bbc/gel-foundations` for spacing and for GEL Typography implemented in Styled Components.
+
+## Installation
+
+`npm install @bbc/psammead-section-divider`
+
+## Props
+
+| Argument  | Type | Required | Default | Example |
+| --------- | ---- | -------- | ------- | ------- |
+| dir | string | no | 'ltr' | 'rtl' |
+| script    | object | yes | latin | { canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, }|
+
+## Usage
+
+```jsx
+import SectionDivider from '@bbc/psammead-section-divider';
+import { latin } from '@bbc/gel-foundations/scripts';
+
+const WrappingComponent = () => <SectionDivider script={latin} dir="ltr">Text here</SectionDivider>;
```

Comment:
Is the recommended usage of this component to be with a heading element? for example, with an `<h2>`? If so, it would be good for the sample usage be using this. e.g.

```jsx
import SectionDivider from '@bbc/psammead-section-divider';
import { SubHeading } from '@bbc/psammead-headings';
import { latin } from '@bbc/gel-foundations/scripts';

const Wrapper = () => (

const WrappingComponent = () => (
  <SectionDivider script={latin}>
    <SubHeading script={latin}>Some subheadline</SubHeading>
  </SectionDivider>
);
```

Also, is it necessary for there to be a `dir="ltr"` on this SectionDivider element? I believe just having it further up the page should be fine.  The reason I'm noting this, is because often people copy and paste the Readme example into their code, so it's good to make sure we have examples that people can clearly use immediately.

Reply:
> Is the recommended usage of this component to be with a heading element? 

No, it isn't. This component puts the children it receives (and as the prop-types make clear, it should only be passed a string) in its own `<h2>`, and intentionally not the SubHeading component, as it uses Double Pica sizing, rather than Trafalgar as in the SubHeading. I have made this more clear on the readme. I was

> Also, is it necessary for there to be a `dir="ltr"` on this SectionDivider element? I believe just having it further up the page should be fine. The reason I'm noting this, is because often people copy and paste the Readme example into their code, so it's good to make sure we have examples that people can clearly use immediately.

No it's not required, but it is valid, the example would work, and IMO helps to remind someone who is copy-and-pasting that, unlike the majority of other components, the styling won't 'just work' for rtl text and whether the prop is being set needs to be considered.